src/lib/y2storage/disk_size.rb

```diff
@@ -33,6 +33,15 @@ module Y2Storage
   # allowed in principle but the special value -1 is reserved to mean
   # 'unlimited'.
   #
+  # {DiskSize} objects are used through {Y2Storage} whenever a size has to be specified.
+  # Notable exceptions here are{Region#start} and {Region#length} in class {Region}, which are +Integer+.
```

Comment:
ah, not last, but end - https://github.com/yast/yast-storage-ng/blob/aacfb73ee25b43c6c6a415c83622ef1f4785d9a6/src/lib/y2storage/region.rb#L59
which should be also integer.